errors/install-sharp.md

```diff
@@ -2,27 +2,24 @@
 
 #### Why This Error Occurred
 
-Using Next.js' built-in Image Optimization requires that you install `sharp`.
+Using Next.js' built-in [Image Optimization](https://nextjs.org/docs/basic-features/image-optimization) requires [sharp](https://www.npmjs.com/package/sharp) as a dependency.
 
-Since `sharp` is optional, it may have been skipped if you installed `next` with the [`--no-optional`](https://docs.npmjs.com/cli/install) flag or it may have been skipped if your platform does not support `sharp`.
+You are seeing this error because your OS was unable to [install sharp](https://sharp.pixelplumbing.com/install) properly, either using pre-built binaries or building from source.
 
 #### Possible Ways to Fix It
 
-Option 1: Install the `sharp` package in your project.
+Option 1: Use a different version of Node.js and try to install `sharp` again.
```

Comment:
Had this error and this worked for me! https://github.com/vercel/next.js/issues/18347#issuecomment-719494101
Maybe instead of saying "use a different version", it should say "update Node.js to the latest stable version"? Not sure just a suggestion :+1:

Reply:
The purpose of this solution is to avoid building from source which is more likely to fail than pre-built binaries.

So in that case, "latest" can actually be the wrong solution when you have to downgrade Node because `sharp` doesn't have pre-built binaries for a new release.

Reply:
Oh ok got it